src/app/Common/Services/Domain/Interfaces/IUserService.cs

```diff
@@ -1,18 +1,25 @@
-﻿using System.Threading.Tasks;
-using Api.Core.Services.Document.Models;
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Common.Dal;
 using Common.Dal.Documents.User;
 using Common.Dal.Repositories;
-using Common.Services.Interfaces;
+using Common.Services.Domain.Models;
 
-namespace Api.Core.Services.Interfaces.Document
+namespace Common.Services.Domain.Interfaces
 {
     public interface IUserService : IDocumentService<User, UserFilter>
     {
+        Task<User> FindByEmailAsync(string email);
+        Task<Page<User>> FindPageAsync(
```

Comment:
Feel like `FindPageAsync` method might be rather common thing for many domain services. Not just `UserService`. Suggest to move to `IDocumentService` and `BaseDocumentService`.